src/wp-includes/template.php

```diff
@@ -711,7 +711,25 @@ function locate_template( $template_names, $load = false, $require_once = true,
 	}
 
 	if ( $load && '' !== $located ) {
+		/**
+		 * Fires before the located template is loaded
```

Comment:
```suggestion
		 * Fires before the located template is loaded.
```

WP documentation standards are to include a period after the description.

:1234: Applies to the description for the after hook too.